### ISSUE TITLE
Enable virtual level pop ups even if virtual levels are turned off

### ISSUE THREAD

**Issue by runeliteissuesmaker** [Nightfirecat/plugin-hub-plugins#5]
Would be nice to not have to use the virtual levels plugin to get the virtual levels popups. 

I dont want want to see the virtual levls on the stats tab, but I do want the notifications for getting a virtual level

**Comment by Nightfirecat** [comment]
These two features you describe are in completely separate plugins. Virtual levels showing on your skills tab is part of the main client's `Virtual levels` plugin, whereas the virtual level up dialogs are part of this `virtual-level-ups` plugin. Turn on whichever you want to actually use.

**Comment by runeliteissuesmaker** [comment]
I had the plugin enabled when getting 118 farming and didn't get a popup, I assumed it was due to the other virtual levels plugin being disabled.

When i got 111 strength I had the virtual levels plugin enabled, and I did get the popup, so I assumed they were connected.

Its pretty infrequent to get a virtual level, so I couldn't really test it, but i just got a popup on my alt for 103 range without the other virtual levels disabled.

I guess I just needed to restart the client or something, but its working fine for me now, thanks for the good plugin!
